docs/router-tutorial/react-navigation-stack/react-navigation-stack.md

```diff
@@ -0,0 +1,338 @@
+---
+title: react-navigation-stack
+parent: React Navigation
+nav_order: 1
+has_children: false
+---
+
+# **React Navigation**
+
+React Navigation (RN) - related libraries that help you move between screens.
+
+## **react-navigation-stack**
+
+It's a concept that can be easily understood by anyone who's ever tried the web. Routing on the web is Stack of RN.
```

Comment:
RN => React Navigation

Reply:
Stack is the little bit different with routing. So `Routing on the web is the role of React Navigation in React Native.` will be better.